### PR TITLE
Add MRP functionality

### PR DESCRIPTION
**Pull request by ckeehan**
Add MRP functionality and update documentation with changes

### CODE REVIEW

File: pyatv/mrp/__init__.py [hunk 1]
```diff
@@ -48,6 +53,7 @@ def __init__(self, loop, protocol):
         if lookup:
             await self.protocol.send(
                 messages.send_hid_event(lookup[0], lookup[1], True))
+            time.sleep(lookup[2])
```
Comment (postlund) [inline pyatv/mrp/__init__.py]: Two things here:
* Why is this needed?
* You must use the asyncio version (asyncio.sleep), this will block the event loop

Comment (ckeehan) [reply pyatv/mrp/__init__.py]: Good catch. I just committed a change to implement asynchronous sleep.

The messages.send_hid_event(arg, arg, True) sends the keydown event to Apple TV, messages.send_hid_event(arg, arg, False) sends keyup. For normal use you want these events to execute back to back with no delay, but a sleep allows for sending "press and hold" button events. Specifically implemented here with the home_hold command.

Comment (postlund) [reply pyatv/mrp/__init__.py]: Ah yes, that is correct. I admit that I did not read my own code good enough here... But it looks good to me!

**Comment by postlund** [comment]
Travis has some complaints, just fix those and it should be ok to merge. I guess you just have to copy from interface to dmap more or less.

**Comment by ckeehan** [comment]
Sorry for the delay. I believe those commits take care of the changes necessary.

**Comment by postlund** [comment]
No worries, do you know if the commands in the DMAP-implementation actually works? Otherwise I was just thinking they could throw NotSupportedError for now.

**Comment by ckeehan** [comment]
Excellent point. Unfortunately I have no way of testing DMAP so I've updated it to throw a NotSupportedError

**Comment by postlund** [comment]
Looks good! 👍
